### ISSUE TITLE
Shapepath missing some values 

### ISSUE THREAD

**Issue by vlakken** [BohemianCoding/SketchAPI#393]
Hi

I am looking at writing a linter plugin with some basic rules but have come across some issues.

Initially I was using the json from the sketch file as my source of truth however, to make it more user friendly I would like to use the javascript api to make a plugin.

If I dump out the details of the objects in the javascript api it appears that some things are missing, for example If I make a simple shape with a border radius I expected a 'points' object to be defined but this is not the case when using the javascript api, in the docs (https://developer.sketchapp.com/reference/api/#shapepath)  it states that a Shapepath has a points value but i am guessing that this is for creating new items.

Here is a screenshot that compares the output form the javascript api and the sketch file json:

https://cl.ly/368ec931db30

To summarise I believe that not all the properties are parsed in the javascript api, and the only way true source of truth for me would be the sketch file json.

**Comment by mathieudutour** [comment]
What version of Sketch are you using? Support for points was added in 53
